news/5581.build.rst

```diff
@@ -0,0 +1,5 @@
+Update ``crossosx`` Vagrant VM to build the toolchain from ``Command Line
+Tools for Xcode`` instead of full ``Xcode package``.
+
+Update ``clang`` in ``linux64`` Vagrant VM to ``clang-11`` from
+``apt.llvm.org`` so it can build ``universal2`` macOS bootloader.
```

Comment:
Oh, I've learnt how to do multiple changes for the same PR and category so that towncrier renders it properly. Split it into two files and call one `5581.build.0.rst` and the other `5581.build.1.rst`.

Reply:
Split both `build` and `feature` entries into separate files.